internal/runners/use/show.go

```diff
@@ -39,8 +39,13 @@ func (s *Show) Run() error {
 		return locale.NewInputError("err_use_show_no_default_project", "No default project is set.")
 	}
 
-	proj, err := project.FromPath(projectDir)
+	proj, err := findproject.FromPath(projectDir, nil)
```

Comment:
This should use `project.FromPath`, not `findproject`. `findproject` is for if the question about the location is a little fuzzy, in this case we know exactly where the project should be.